sdk/avalon_sdk/ethereum/ethereum_wrapper.py

```diff
@@ -193,8 +197,17 @@ def get_contract_instance(self, contract_file_name, contract_address):
         return self.__w3.eth.contract(address=contract_address,
                                       abi=contract_interface["abi"])
 
+    def get_contract_instance_from_json(self, json_file_name,
+                                        contract_address):
```

Comment:
Add a comment that this json file is compiled solidity file and is not used currently